py/selenium/webdriver/common/action_chains.py

```diff
@@ -203,7 +208,7 @@ def key_down(self, value, element=None):
 
         return self
 
-    def key_up(self, value, element=None):
+    def key_up(self, value, element=Optional[WebElement]) -> "ActionChains":
```

Comment:
`: Optional[WebElement] = None`

same above

Reply:
also missing type for the `value` arg

Reply:
Fixed.